tasks/build.py

```diff
@@ -474,6 +475,12 @@ def prepare_jobs(pr, cfg, event_info, action_filter):
     #      call to just before download_pr
     year_month, pr_id, run_dir = create_pr_dir(pr, cfg, event_info)
 
+    accelerator = "none"
+    # determine accelerator from action_filter argument
+    accelerators = action_filter.get_filter_by_component(tools_filter.FILTER_COMPONENT_ACCEL)
+    if len(accelerators) > 0:
+        accelerator = accelerators[0]
```

Comment:
Should we warn or something when the list has more than 1 element?

Reply:
Yep. Logging is maybe sufficient.

Reply:
Done in https://github.com/EESSI/eessi-bot-software-layer/pull/280/commits/8f44ba9846c7d9eb37fefd232e0884ab9651e627

Also logs in case there is no element.